apps/arweave/src/ar_peers.erl

```diff
@@ -34,23 +37,47 @@
 %% The maximum number of peers to return from get_peers/0.
 -define(MAX_PEERS, 1000).
 
-%% The number of failed requests in a row we tolerate before dropping the peer.
--define(TOLERATE_FAILURE_COUNT, 20).
+%% Minimum average_success we'll tolerate before dropping a peer.
+-define(MINIMUM_SUCCESS, 0.8).
+-define(THROUGHPUT_ALPHA, 0.05).
+%% set so that roughly 20 consecutive failures will drop the average_success below 0.5
+-define(SUCCESS_ALPHA, 0.035). 
```

Comment:
It should be approximately the average over 1 / 0.035 = 28.5 data points so considering we had successes before a series of consecutive failures, 28.5 / 2 failures are sufficient to bring the average down to 50%. Does it sound correct?

Reply:
The math is weird because of how ema is calculated - I'll try to find the equation again. But when I ran it 0.035 ended up being 20 failures to bring value from 1 to 0.5